examples/stable/mssql/README.md

```diff
@@ -0,0 +1,353 @@
+
+# Create database "TestDB"
+1> CREATE DATABASE TestDB
+2> SELECT Name from sys.Databases
+3> GO
+Name                                                                                                                            
+---------------------------------------------------------------------------
+master                                                                                                                          
+tempdb                                                                                                                          
+model                                                                                                                           
+msdb                                                                                                                            
+TestDB                                                                                                                          
+
+# Create table "Inventory" inside database "TestDB"
+1> USE TestDB
+2> CREATE TABLE Inventory (id INT, name NVARCHAR(50), quantity INT)
+3> INSERT INTO Inventory VALUES (1, 'banana', 150); INSERT INTO Inventory VALUES (2, 'orange', 154);
+4> GO
+
+(1 rows affected)
+
+(1 rows affected)
+
+# View data in "Inventory" table
+1> SELECT * FROM Inventory;
+2> GO
+id          name                                               quantity   
+----------- -------------------------------------------------- -----------
+          1 banana                                                     150
+          2 orange                                                     154
+
+```
+After following all given steps database named `TestDB` should have tabled called `Inventory`
+with some data.
+
+## Integrating with Kanister
+
+If you have deployed SQL Server with name other than `mssql-deployment` and namespace other than `default`, 
+you need to modify the commands(backup, restore and delete) used below to use the correct release name and namespace
+
+### Create Profile
+Create Profile CR if not created already
+
+```bash
+$ kanctl create profile s3compliant --access-key <aws-access-key-id> \
+	--secret-key <aws-secret-key> \
+	--bucket <s3-bucket-name> \ 
+	--region <region-name> \
+	--namespace kanister
+```
+You can read more about the Profile custom Kanister resource [here](https://docs.kanister.io/architecture.html?highlight=profile#profiles).
+
+**NOTE:**
+
+The above command will configure a location where artifacts resulting from Kanister
+data operations such as backup should go. This is stored as a `profiles.cr.kanister.io`
+*CustomResource (CR)* which is then referenced in Kanister ActionSets. Every ActionSet
+requires a Profile reference to complete the action. This CR (`profiles.cr.kanister.io`)
+can be shared between Kanister-enabled application instances.
+
+### Create Blueprint
+
+Create Blueprint in the same namespace as the Kanister controller
+
+Execute following command to create blueprint
```

Comment:
```suggestion
Execute the following command to create the blueprint
```

Reply:
`I think execute following command to create the blueprint` sounds better